core/src/main/java/org/elasticsearch/index/query/PrefixQueryParser.java

```diff
@@ -30,7 +30,9 @@
  */
 public class PrefixQueryParser implements QueryParser<PrefixQueryBuilder> {
 
-    private static final ParseField NAME_FIELD = new ParseField("_name").withAllDeprecated("query name is not supported in short version of prefix query");
+    public static final ParseField NAME_FIELD = new ParseField("_name").withAllDeprecated("query name is not supported in short version of prefix query");
```

Comment:
+1 on renaming.

According to "git praise" @javanna you originally introduced the NAME_FIELD - not sure if you had a closer look at this change when going through the PR last time you commented.